pkgs/test_core/lib/src/runner/configuration/args.dart

```diff
@@ -105,6 +106,10 @@ final ArgParser _parser = (() {
       help: "Don't re-run tests that have retry set.",
       defaultsTo: false,
       negatable: false);
+  parser.addOption("test-randomize-ordering-seed",
```

Comment:
Maybe make mention that we only accept a range of values.

Reply:
Adding "(must be a 32bit unsigned integer)"